content/community/software.md

```diff
@@ -1,6 +1,6 @@
 +++
 type = "article"
-title = "Software Sites"
+title = "Software Repositories"
```

Comment:
This list things that are clearly not repositories: bebytes and my BeOS archive at least.